CMakeLists.txt

```diff
@@ -402,6 +402,15 @@ if(TS_USE_HWLOC)
   )
 endif()
 
+check_symbol_exists(SO_TXTIME "sys/socket.h" SO_TXTIME_FOUND)
+set(CMAKE_EXTRA_INCLUDE_FILES "linux/net_tstamp.h")
+check_type_size("struct sock_txtime" STRUCT_SOCK_TXTIME_FOUND)
+unset(CMAKE_EXTRA_INCLUDE_FILES)
+
+if(SO_TXTIME_FOUND AND STRUCT_SOCK_TXTIME_FOUND)
```

Comment:
Are there platforms that don't have either of these? Checking the both is fine but I'm just curious.

Reply:
I did check on FreeBSD, `net_tstamp.h` wasn't  available, neither was `SO_TXTIME` .  Are they available in Mac?

Reply:
No, macOS doesn't have neither of the two. I agree that checking the both is more robust. I just wondered what are used if there's a platform that has only one of the two.